examples/capi-quick-start/aws-cluster-ami.yaml

```diff
@@ -27,7 +27,8 @@ spec:
               providers:
                 - name: aws-ebs
             nfd: {}
-          aws: {}
+          aws:
+            region: us-west-2
```

Comment:
I saw this new example, what is this file?

Reply:
this file was added to main with Jimmi's PR about using `unstructured`. We can remove this file in this PR and add `ami` spec to the existing `aws-cluster.yaml`

Reply:
oh sorry, we can remove this, didn't mean to add it - i was using it for testing.